R/test-files.R

```diff
@@ -148,9 +148,9 @@ test_files <- function(test_dir,
 
 test_one_file <- function(path, env = test_env()) {
   reporter <- get_reporter()
-  on.exit(teardown_run(), add = TRUE)
+  on.exit(teardown_run(tempdir()), add = TRUE)
```

Comment:
This feels iffy to me, but it's what preserved the existing behaviour in the tests.